x-pack/test_serverless/functional/page_objects/svl_search_inference_management_page.ts

```diff
@@ -27,53 +27,22 @@ export function SvlSearchInferenceManagementPageProvider({ getService }: FtrProv
 
         const table = await testSubjects.find('inferenceEndpointTable');
         const rows = await table.findAllByClassName('euiTableRow');
-        expect(rows.length).to.equal(2);
-
-        const elserEndpointCell = await rows[0].findByTestSubject('endpointCell');
-        const elserEndpointName = await elserEndpointCell.getVisibleText();
-        expect(elserEndpointName).to.contain('.elser-2-elasticsearch');
-
-        const elserProviderCell = await rows[0].findByTestSubject('providerCell');
-        const elserProviderName = await elserProviderCell.getVisibleText();
-        expect(elserProviderName).to.contain('Elasticsearch');
-        expect(elserProviderName).to.contain('.elser_model_2');
-
-        const elserTypeCell = await rows[0].findByTestSubject('typeCell');
-        const elserTypeName = await elserTypeCell.getVisibleText();
-        expect(elserTypeName).to.contain('sparse_embedding');
-
-        const e5EndpointCell = await rows[1].findByTestSubject('endpointCell');
-        const e5EndpointName = await e5EndpointCell.getVisibleText();
-        expect(e5EndpointName).to.contain('.multilingual-e5-small-elasticsearch');
-
-        const e5ProviderCell = await rows[1].findByTestSubject('providerCell');
-        const e5ProviderName = await e5ProviderCell.getVisibleText();
-        expect(e5ProviderName).to.contain('Elasticsearch');
-        expect(e5ProviderName).to.contain('.multilingual-e5-small');
-
-        const e5TypeCell = await rows[1].findByTestSubject('typeCell');
-        const e5TypeName = await e5TypeCell.getVisibleText();
-        expect(e5TypeName).to.contain('text_embedding');
+        // we need at least one (ELSER) otherwise index mapping will start to fail
+        expect(rows.length).to.greaterThan(1);
```

Comment:
Should we also check that one of the entries contains an ELSER endpoint?

Reply:
Done